tests/testthat/test-create-soils.R

```diff
@@ -0,0 +1,5 @@
+test_that("create_soils returns appropriate error", {
+  expect_snapshot_error(
```

Comment:
```suggestion
  expect_snapshot(error = TRUE,
```

`expect_snapshot_*()` are no longer recommended. https://testthat.r-lib.org/reference/expect_snapshot_output.html

Reply:
Good point - I didn't realise that! Their status is "questioning", which doesn't mean superseded or deprecated, but it probably is a good idea to future proof.

Reply:
`expect_snapshot(error = TRUE` was causing the test to [fail the R CMD GHA checks](https://github.com/WA-Department-of-Agriculture/soils/actions/runs/7704088127/job/20995716153#step:6:170) (even though it passed locally on my machine). I reverted back to `expect_snapshot_error` and the [test now passes](https://github.com/WA-Department-of-Agriculture/soils/actions/runs/7704291268/job/20996313874#step:6:177).